public/theme/boltcms-2021/page.twig

```diff
@@ -0,0 +1,35 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+    <main>
+        <div class="subheader">
+            <h1>
+                {{ record|title }}
+            </h1>
+            <p>
+                {{ record.subtitle }}
+            </p>
+        </div>
+        {% if record.image is not empty %}
+            <img class="fluid" src="{{ record.image|thumbnail(800, 600) }}" alt="{{ record.image.alt|default }}">
+        {% endif %}
+
+        {# Ugly fix because right now allow_twig does not work in article. #}
+        {% if '_include_github_contributors' in record.content|raw %}
+            {% set contributors %}
+               {% include 'partials/_contributors.twig'%}
+            {% endset %}
+
+            {{ record.content|replace({'_include_github_contributors': contributors})|raw }}
+        {% else %}
+            {{ record.content }}
+        {% endif %}
+
+        {{ boltform(record.slug, warn=false) }}
```

Comment:
Ah, good that this is here: we need to test if the new design shows HTML5 forms well 🥇

Reply:
?

Reply:
https://boltcms.io/submit-showcase <- this form :-)

Reply:
We wait for css from Derk